packages/devtools_app/pubspec.yaml

```diff
@@ -43,6 +43,8 @@ dependencies:
   mime: ^1.0.0
   path: ^1.8.0
   provider: ^6.0.2
+  # Only used for debug mode logic.
+  shared_preferences: ^2.0.15
```

Comment:
couple questions 
- can this be moved to dev_dependencies?
- will this persist only for a single instance of the macos runner or chrome? or does it persist if you do a full stop app and re-run?

Reply:
I don't think it can be moved to dev dependencies as it is part of the main package and I don't know an easy way to move it outside the main package short of having a wacky debugging entry point for DevTools outside the lib directory.

Reply:
persists even if you do a full-stop and re-run. personally I found that useful as sometimes Flutter gets in a bad state or I'm paranoid it is in a bad state so I full stop. We could store the time-stamp of the last run and only reuse the setting if it is less than an hour old or something to reduce the # of annoying cases.